src/rospkg/rospack.py

```diff
@@ -391,6 +392,36 @@ def stack_of(self, package):
             else:
                 d = os.path.dirname(d)
 
+    def get_licenses(self, pkg_name, implicit=True):
+        """
+        @summary: Return a list of licenses and the packages in the dependency tree
+            for the given package. Special value 'ERR' is used as the license for the
```

Comment:
I think this documentation line is out of date since `None` is now used right?
```suggestion
            for the given package. ``None`` is used as the license for the
```

Reply:
TLDR; I now set the string "`license_not_found`" for the packages that any license wasn't detected of.

Follow-up for https://github.com/ros-infrastructure/rospkg/pull/133#discussion_r270967801, yes I followed the review comment and changed to 'None'. Today (on Ubuntu 20.04 with Python 3.8.5) I found it causes "`TypeError ('<' not supported between instances of 'NoneType' and 'License`"), and found that having `None` as the dict key caused that. Thus I switched back to using string. This time I'm trying to make the variable name more descriptive so hope it got better.